include/mbedtls/config.h

```diff
@@ -3671,6 +3671,25 @@
  */
 //#define MBEDTLS_PSA_HMAC_DRBG_MD_TYPE MBEDTLS_MD_SHA256
 
+/** \def MBEDTLS_PSA_MAX_KEY_BITS
+ * Restrict the PSA Library to supporting a maximum key size (in bits).
+ *
+ * If this option is unset, the library will fall back to a default value of
+ * 8191 bytes = 65528 bits.
+ */
+//#define MBEDTLS_PSA_MAX_KEY_BITS 0xfff8
```

Comment:
Done both. Chose `psa_crypto.c` because there were already instances of static assertion in there, so it made sense to apply this one the same way. It's located together with the 'import key to slot' function, which I think is the primary location where the key length would get checked.